libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -76,6 +77,14 @@ const AP_Param::GroupInfo HarmonicNotchFilterParams::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("MODE", 7, HarmonicNotchFilterParams, _tracking_mode, 1),
 
+    // @Param: OPTS
+    // @DisplayName: Harmonic Notch Filter options
+    // @Description: Harmonic Notch Filter options.
+    // @Bitmask: 0:Double notch
+    // @User: Advanced
+    // @RebootRequired: True
+    AP_GROUPINFO("OPTS", 8, HarmonicNotchFilterParams, _options, 0),
```

Comment:
Can we call it "OPTIONS" instead or would this make the parameter name too long?  We've normally named these types of parameters _OPTIONS..

Reply:
It's prefixed with INS_HNTCH_ so OPTIONS makes it just too long (by one char I think). Other suggestions?